orchestrator/orchestrator.go

```diff
@@ -273,6 +274,7 @@ func (s *Orchestrator) processTask(nodeKey string, task *process.Process_Node_Ta
 		TaskKey:      task.TaskKey,
 		Inputs:       data,
 		ExecutorHash: executor.Hash,
+		Price:        s.execPrice,
```

Comment:
I think orchestrator shouldn't be responsible anyhow for setting the min price. Espessialy if the executor is chosenen randomly from runners list.

Reply:
This will be the responsibility of the orchestrator because it is the one triggering the execution. After I agree now it's really simple and the price calculation should be much more complex and not only a constant. This price will be calculated based on the price defined by the service/runner.